docs/release_note_generator/main.py

```diff
@@ -0,0 +1,71 @@
+# Usage:
```

Comment:
I wonder if there are other existing tools that basically do this job instead of us maintaining our own script. Have we looked at 3rd party solutions?

It could be useful to use 3rd party solutions if they add new extensions / reduce our maintenance burden.